### PR TITLE
feat: fix search render issue Hub page and add deboucing to search

### PR DESCRIPTION
**Pull request by iamnamananand996**
Because

- fix search render issue Hub page

This commit

- fix search render issue Hub page

### CODE REVIEW

File: packages/toolkit/src/view/hub/hub-view/Body.tsx [hunk 1]
```diff
@@ -105,30 +121,36 @@ export const Body = ({
                 <Icons.SearchSm className="my-auto h-4 w-4 stroke-semantic-fg-primary" />
               </Input.LeftIcon>
               <Input.Core
-                value={searchCode ?? ""}
+                value={searchValue ?? ""}
```
Comment (EiffelFly) [inline packages/toolkit/src/view/hub/hub-view/Body.tsx:124]: @iamnamananand996 Let's make this state more clearer. Let call it searchInputValue

**Review by EiffelFly — COMMENTED** [review]
Hi @iamnamananand996 I observe multiple issues on this PR

- You didn't include the pnpm-lock change in this PR which cause nearly all the test failed to build the console image
- We have the same search functionality on /pipelines page, please help us include debounce in there too
- I left some suggestion

File: packages/toolkit/src/view/hub/hub-view/Body.tsx (hunk 1 above)
Comment (iamnamananand996) [reply packages/toolkit/src/view/hub/hub-view/Body.tsx]: Hi @EiffelFly, update all suggested changes

**Comment by EiffelFly** [comment]
@iamnamananand996 When you submit the PR, please make sure at least "pnpm test" and "pnpm lint" is passed

**Comment by iamnamananand996** [comment]
> pnpm lint

Hi @EiffelFly, should good now

**Pull request merged**